dashboard/apps/dashboard/views.py

```diff
@@ -124,3 +131,87 @@ def sync_from_float(request):
     return JsonResponse({
         'status': 'STARTED'
     })
+
+
+class PortfolioExportView(View):
+    http_method_names = ['get']
+
+    def format(self, value):
+        if isinstance(value, date):
+            return value.strftime('%d/%m/%Y')
+        return value
+
+    def get(self, request, *args, **kwargs):
+        show = kwargs.get('show', 'visible')
+        now = datetime.now()
+        fname = '%s_%s_%s.xlsx' % (
+            'ProductData',
+            show,
+            now.strftime('%Y-%m-%d_%H:%M:%S'))
+
+        date_style = Style(number_format='DD/MM/YYYY')
+        bold_font = Font(bold=True)
+        bold_style = Style(font=bold_font)
+        currency_style = Style(number_format='£#,##0.00')
+
+        fields = (
+            ('Id', None),
+            ('Name', None),
+            ('Description', None),
+            ('Area name', None),
+            ('Discovery date', date_style),
+            ('Alpha_date', date_style),
+            ('Beta date', date_style),
+            ('Live date', date_style),
+            ('End date', date_style),
+            ('Discovery fte', None),
+            ('Alpha fte', None),
+            ('Beta fte', None),
+            ('Live fte', None),
+            ('Final budget', currency_style),
+            ('Cost of discovery', currency_style),
+            ('Cost of alpha', currency_style),
+            ('Cost of beta', currency_style),
+            ('Cost in FY 14-15', currency_style),
```

Comment:
`Cost in FY 14-15`, `Cost in FY 15-16`, etc should be de-hardcoded. essentially it's just a time window including last 2 financial years, this financial year and next financial year. 

if a dictionary is used, more information can be encapsulated including the kwargs to the function call.